From 351fcce4e3246c7605b372fb9217248a8df987ba Mon Sep 17 00:00:00 2001
From: Jeff Walsh <fejfighter@gmail.com>
Date: Mon, 19 Sep 2022 10:26:23 +1000
Subject: [PATCH] No error on warning

---
 autogen.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/autogen.sh b/autogen.sh
index dc68516..52fca3c 100755
--- a/autogen.sh
+++ b/autogen.sh
@@ -24,8 +24,7 @@ for i in $files; do rm -f -- "$i.$$.tmp"; done
 trap cleanup $signals
 for i in $files; do mv -f -- "$i" "$i.$$.tmp"; done
 
-autoreconf -v -f -i --warnings=all,error
+autoreconf -v -f -i --warnings=all
 status=$?
 
 rm -rf autom4te.cache
--
2.37.1

